audio_service/android/src/main/java/com/ryanheise/audioservice/AudioService.java

```diff
@@ -521,7 +521,7 @@ private void updateNotification() {
     }
 
     private boolean enterPlayingState() {
-        startService(new Intent(AudioService.this, AudioService.class));
+        startForegroundService(new Intent(AudioService.this, AudioService.class));
```

Comment:
Can we use `ContextCompat.startForegroundService` instead? This will allow it to work down to the min SDK version.